recipes/tomlplusplus/all/conanfile.py

```diff
@@ -73,3 +79,6 @@ def package_info(self):
         self.cpp_info.set_property("cmake_target_name", "tomlplusplus::tomlplusplus")
         self.cpp_info.bindirs = []
         self.cpp_info.libdirs = []
+        if self.options.exceptions.value is not None:
+            define_value = "1" if self.options.exceptions is True else "0"
+            self.cpp_info.defines.append(f"TOML_EXCEPTIONS={define_value}")
```

Comment:
Seems like upstream nicely handles the case where this is hardcoded to 1 but the compiler does not have exception support enabled :)